python_bindings/tests/end-to-python-end-test/two_player_game_test.py

```diff
@@ -0,0 +1,572 @@
+                                   
+-1                                
+                                    
+    
+"""
+
+
+def check_third_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=1, y=0),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.TopLeftEdge,
+        featureType=FeatureType.Field,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 0])
+
+    return game_id, game
+
+
+"""
+// T cross road
+// player2 places meeple (@) on road going down
+            -1    0    1    2    3
+            
+            
+                ..........          
+                ..........          
+1               --1--@-2..         
+                ./ \...|..          
+                |   |..|..          
+           ..|..|   |..|..          
+           ..|...\ /.!.|..          
+0          ..4----0----3..          
+           ..|............          
+           ..@............          
+                                    
+                                   
+-1                                
+                                    
+                        
+"""
+
+
+def check_fourth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=-1, y=0),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Bottom,
+        featureType=FeatureType.Road,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 0])
+
+    return game_id, game
+
+
+"""
+// monastery with single road
+// player1 places meeple (!) on a monastery
+// road from 4 to 5 is finished, so player2 scores 2 points
+            -1    0    1    2    3
+            
+            
+                ..........          
+                ..........          
+1               --1--@-2..         
+                ./ \...|..          
+                |   |..|..          
+           ..|..|   |..|..          
+           ..|...\ /.!.|..          
+0          ..4----0----3..          
+           ..|............          
+           ..|............          
+           ..|..                    
+           .[ ].                   
+-1         .[5].                  
+           .[!].                    
+           .....            
+"""
+
+
+def check_fifth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=-1, y=-1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.Monastery,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 2])
+
+    return game_id, game
+
+
+"""
+// Two city edges not connected
+// player 2 places meeple(@) on the right city
```

Comment:
```suggestion
// player 2 places meeple (@) on the right city
```

other comments in this file have a space before the meeple sign

Reply:
fixed